scripts/process_uploads.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+"""
+Processes files uploaded to gcp
```

Comment:
Can we make this a little more specific? :)

Reply:
Sure! Hopefully, the new revision is clearer.